packages/x-codemod/src/v6.0.0/data-grid/row-selection-props-rename/index.ts

```diff
@@ -8,6 +8,7 @@ const props = {
   disableMultipleSelection: 'disableMultipleRowSelection',
   showCellRightBorder: 'showCellVerticalBorder',
   showColumnRightBorder: 'showColumnVerticalBorder',
+  headerHeight: 'columnHeaderHeight',
```

Comment:
Nitpick: We could also consider delivering such improvements via a separate PR, it would also help with increased visibility and discoverability if anyone is following the changelog. 😉

Reply:
This specific change fell under same category which was previously covered by a codemod and was a minor one, so I pushed it along with this one, but you are right, delivering under separate ones might be more helpful. 👍

Reply:
I was thinking about both this change as well as the `GridColumnMenuItemProps` rename. 😉

Reply:
Ah ok, you are right actually 😄 
Let me split this into respective ones

Reply:
No pressure. We are still on initial beta releases, it's not that the package is widely used. 🙈

Reply:
No no, it's a good idea. It'll be easier for us too to track the changes this way.
Thanks for the useful Nitpick 😉

Reply:
Split respective changes into https://github.com/mui/mui-x/pull/7710 and https://github.com/mui/mui-x/pull/7711